waspc/test/Util/FilePathTest.hs

```diff
@@ -0,0 +1,19 @@
+module Util.FilePathTest where
+
+import Test.Tasty.Hspec (Spec, describe, it, shouldBe)
+import Wasp.Util.FilePath (removePathPrefix)
+
+spec_FilePathTest :: Spec
+spec_FilePathTest = do
+  describe "removePathPrefix" $ do
+    it "removes path prefix from a path that includes that prefix" $ do
+      removePathPrefix "src/" "src/Component.js" `shouldBe` Just "Component.js"
+
+    it "removes path prefix even if the prefix doesn't have trailing slash" $ do
```

Comment:
Oh yeah this is also important! Otherwise we would turn path on the right into absolute path. Good one!